frontend/src/components/Pages/InspectionPage/InspectionUtilities.tsx

```diff
@@ -1,6 +1,6 @@
 import { Card, Typography } from '@equinor/eds-core-react'
 import styled from 'styled-components'
-import { DeckMissionCount, Inspection } from './InspectionSection'
```

Comment:
Nothing else is using DeckMissionCount so it can be deleted in inspectionSection